src/modules/Aieracast/index.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { Fragment, ReactElement, RefObject, useCallback, useEffect, useRef, useState } from 'react';
+import './styles.css';
+import { EventList, EventRowProps } from '../EventList';
+import { Transcript } from '../Transcript';
+import { getEventCreatorName, getPrimaryQuote } from '@aiera/client-sdk/lib/data';
+import { Toggle } from '@aiera/client-sdk/components/Toggle';
+import { User } from '@aiera/client-sdk/types';
+import { DateTime } from 'luxon';
+import classNames from 'classnames';
+import { isToday } from '@aiera/client-sdk/lib/datetimes';
+import { Tooltip } from '@aiera/client-sdk/components/Tooltip';
+import { prettyLineBreak } from '@aiera/client-sdk/lib/strings';
+import { TimeAgo } from '@aiera/client-sdk/components/TimeAgo';
+import { Playbar } from '@aiera/client-sdk/components/Playbar';
+import { Chevron } from '@aiera/client-sdk/components/Svg/Chevron';
+
+interface AieracastSharedProps {}
+
+/** @notExported */
+interface AieracastUIProps extends AieracastSharedProps {
+    openEventIds: string[];
+    toggleEvent: (id: string) => void;
+    scrollRef: RefObject<HTMLDivElement>;
+}
+
+export function AieracastUI(props: AieracastUIProps): ReactElement {
+    const { openEventIds, toggleEvent, scrollRef } = props;
+
+    const EventRow = ({
+        customOnly,
+        event,
+        numMentions,
+        maxHits = 0,
+        isRefetching,
+        refetch,
+        renderedRefetch,
+        searchTerm,
+        showDivider,
+    }: EventRowProps) => {
+        const hitRatio = (numMentions || 0) / maxHits;
```

Comment:
Can't divide by 0, so if `maxHits` could be undefined, maybe `maxHits ? (numMentions || 0) / maxHits : 0`